_posts/2023-05-15-May-Update-Finishing-my-Second-Igalia-CE.md

```diff
@@ -0,0 +1,297 @@
+For example, the `kms_plane_scaling` test was failing in VKMS, because it didn't consider the case in which the driver did not have the rotation property.
+As VKMS didn't use to have the rotation property, the tests were failing instead of skipping.
+Therefore, I just developed a path for drivers without the rotation property for the tests to skip.
+
+I sent improvements to the `kms_plane_scaling`, `kms_flip`, and `kms_plane` tests, making the tests pass or skip on all cases for the VKMS.
+
+| Patches | Status |
+| :---- | :----: |
+| [[PATCH i-g-t] tests/kms_plane_scaling: negative tests can return -EINVAL or -ERANGE](https://patchwork.freedesktop.org/patch/533643/?series=116907&rev=1) | **Accepted** |
+| [[PATCH i-g-t] tests/kms_plane_scaling: fix variable misspelling](https://patchwork.freedesktop.org/patch/533645/?series=116907&rev=1)                     | **Accepted** |
+| [[PATCH i-g-t] tests/kms_plane_scaling: remove unused parameters](https://patchwork.freedesktop.org/patch/533644/?series=116907&rev=1)                     | **Accepted** |
+| [[PATCH i-g-t] tests/kms_plane_scaling: Only set rotation if rotation != rotate-0](https://patchwork.freedesktop.org/series/117250/)                       | **Accepted** |
+| [[PATCH v2 i-g-t] tests/kms_flip: Check if is Intel device before doing all the setup](https://patchwork.freedesktop.org/series/117246/)                   | **Accepted** |
+| [[PATCH i-g-t v2] tests/kms_plane: allow pixel-format tests to run on drivers without legacy LUT](https://patchwork.freedesktop.org/patch/533753/)         | In Review |
+
+
+## VKMS CI List
+
+One important thing to VKMS is creating a baseline of tests that should pass.
+This way new contributions can be tested against this baseline and we can avoid introducing regressions in the codebase.
+This is why I sent a patch to IGT to create a testlist for the VKMS driver.
+This is a list of all the KMS tests that must pass on the VKMS driver.
+
+With new features being introduced in VKMS, it is important to make sure that the test list is kept updated.
+So, I sent a couple of updates during my time at the Igalia CE.
+I intend to keep this list updated as long as I can.
+
+| Series | Status |
+| :---- | :----: |
+| [[PATCH i-g-t] tests/vkms: Create a testlist to the vkms DRM driver](https://patchwork.freedesktop.org/series/112972/) | **Accepted** |
+| [[PATCH i-g-t 0/3] tests/vkms: Update VKMS's testlist](https://patchwork.freedesktop.org/series/116316/)               | **Accepted** |
+
+# Acknowledgment
+---
+First, I would like to thank my great mentor [Melissa Wen](https://melissawen.github.io/).
+Melissa and I are completing a year together as mentee and mentor and it has been an amazing journey.
+Since GSoC, Melissa has been helping me by answering every single question I have and providing me with great encouragement.
+I have a lot of admiration for her and I'm really grateful for having her as my mentor during the last year.
```

Comment:
Me too! It was a great experience following your growth and participating in this learning process around GPU development. Happy to see how you evolve fast, with good quality work and commitment.